Specs/DateTime/English/DateTimeModel.json

```diff
@@ -13215,5 +13215,155 @@
         }
       }
     ]
+  },
+  {
+    "Input": "I will handle all the stuff within next week.",
+    "Context": {
+      "ReferenceDateTime": "2019-07-16T00:00:00"
+    },
+    "NotSupported": "javascript, python",
+    "Results": [
+      {
+        "Text": "within next week",
+        "Start": 28,
+        "End": 43,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(2019-07-22,2019-07-29,P1W)",
+              "type": "daterange",
+              "start": "2019-07-22",
+              "end": "2019-07-29"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "I will handle all the stuff within one week.",
+    "Context": {
+      "ReferenceDateTime": "2019-07-16T00:00:00"
+    },
+    "NotSupported": "javascript, python",
+    "Results": [
+      {
+        "Text": "within one week",
+        "Start": 28,
+        "End": 42,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(2019-07-16,2019-07-23,P1W)",
+              "type": "daterange",
+              "start": "2019-07-16",
+              "end": "2019-07-23"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "I will handle all the stuff in next two weeks.",
```

Comment:
Does this pr also supports "I will handle all the stuff within the next two weeks." ? Can you add that as a spec case?